backend/src/bundles/talent-hard-skills/talent-hard-skills.entity.ts

```diff
@@ -0,0 +1,77 @@
+import { type Entity } from '~/common/types/types.js';
+
+class TalentHardSkillsEntity implements Entity {
+    private 'id': string | null;
+
+    private 'hardSkillId': string;
+
+    private 'userDetailsId': string | null;
```

Comment:
Hard-skill assigned to user-details can be null?
```suggestion
    private 'userDetailsId': string;
```

Reply:
according to our migration add_talent_hard_skills_table.ts user-details can be null

Reply:
Can you please explain why userDetailsId can be null ?

Reply:
you are right. I removed null